src/bundle/Resources/views/themes/admin/ui/component/dropdown/dropdown.html.twig

```diff
@@ -68,7 +71,7 @@
 
                             {% include '@ibexadesign/ui/component/dropdown/dropdown_selected_item.html.twig' with {
                                 value: first_choice.value,
-                                label: _self.get_translated_label(first_choice.label),
+                                label: _self.get_translated_label(first_choice.label, false),
```

Comment:
There was no `translation_domain` so I guess we want to pass `false` here (as the condition in macro is `translation_domain is same as(false)`, which is false for not defined `translation_domain`)? 
Can you confirm @GrabowskiM? :)

Reply:
We want to use `translation_domain` variable, as in all other uses of this macro (for example line 67)